ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml

```diff
@@ -0,0 +1,39 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-rosa-4.12-prod.yaml
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-rosa-hypershift-cluster-density
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-perf-qe
+    env:
+      OCM_LOGIN_ENV: production
+      OPENSHIFT_VERSION: "4.12"
+      REGION: us-east-2
+      ROSA_LOGIN_ENV: production
+    test:
+    - ref: openshift-qe-cluster-density
+    workflow: rosa-aws-sts-hypershift
```

Comment:
Still think it would be good to add hypershift or something more specific for this environment that "stable-prod", not very clear from title what it's running

Reply:
The intention of the file name is everything being testing in production with stable release. Can be classic rosa, hypershift, non-rosa tests etc... 
Feel free to suggest a name.

Reply:
Discussed with Paige - we agreed that keeping this file for all testing we do in production is better and hence, name of the file is acceptable.
Tests potentially to be added here:
- cluster-density-v2 for non-rosa, rosa, hypershift - to serve the purpose as smoke test in prod
- reliability tests - non-rosa, rosa, hypershift